spec/models/ext_management_system_spec.rb

```diff
@@ -501,145 +501,6 @@
     end
   end
 
-  context "#pause!" do
-    before do
-      Zone.seed
-    end
-
-    it 'disables an ems with child managers and moves them to maintenance zone' do
-      zone  = FactoryBot.create(:zone)
-      ems   = FactoryBot.create(:ext_management_system, :zone => zone)
-      child = FactoryBot.create(:ext_management_system, :zone => zone)
-      ems.child_managers << child
-
-      2.times do
-        ems.pause!
-
-        expect(ems.enabled).to eq(false)
-        expect(ems.zone).to eq(Zone.maintenance_zone)
-        expect(ems.zone_before_pause).to eq(zone)
-
-        child.reload
-        expect(child.enabled).to eq(false)
-        expect(child.zone).to eq(Zone.maintenance_zone)
-        expect(child.zone_before_pause).to eq(zone)
-      end
-    end
-
-    it 'disables an ems and moves child managers created by ensure_managers() to maintenance zone' do
-      zone  = FactoryBot.create(:zone)
-      emses = {
-        :amazon_cloud    => FactoryBot.create(:ems_amazon,          :zone => zone),
-        :azure_cloud     => FactoryBot.create(:ems_azure,           :zone => zone),
-        :google_cloud    => FactoryBot.create(:ems_google,          :zone => zone),
-        :openstack_cloud => FactoryBot.create(:ems_openstack,       :zone => zone),
-        :openstack_infra => FactoryBot.create(:ems_openstack_infra, :zone => zone),
-        :vmware_cloud    => FactoryBot.create(:ems_vmware_cloud,    :zone => zone),
-        :vpc_cloud       => FactoryBot.create(:ems_ibm_cloud_vpc,   :zone => zone)
-      }
```

Comment:
I always thought it was wrong that we looped through these providers in core versus having shared_examples that each include.  This list doesn't even include all providers, with shared examples we can add a spec test to the provider generator.